<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0" columnminimumwidth="0,0,0,0,0">
   <item row="1" column="1">
    <widget class="QComboBox" name="cbLineStyle">
     <item>
      <property name="text">
       <string>直线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>长虚线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>短虚线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>点线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="spLineWidth">
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>线条</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="txtLabel"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnLineColor">
     <property name="text">
      <string>颜色</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>数据
(以空格或
回车隔开)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QTextEdit" name="txtData"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
